rotkehlchen/tests/api/test_users.py

```diff
@@ -215,8 +237,8 @@ def test_user_creation_with_premium_credentials(rotkehlchen_api_server, data_dir
         PremiumCredentials(VALID_PREMIUM_KEY, VALID_PREMIUM_SECRET),
         username=username,
         patch_get=True,
-        metadata_last_modify_ts=0,
-        metadata_data_hash=b'',
+        metadata_last_modify_ts=Timestamp(0),
+        metadata_data_hash=str(b''),
```

Comment:
If this should be a string then this is what you need to do

```suggestion
        metadata_data_hash='',
```